### PR TITLE
Build docker images in travis.

### PR DESCRIPTION
**Pull request by derrickburns**
This builds Docker images in Travis for each push that passes unit tests.

### CODE REVIEW

File: artifact.sh [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#!/bin/sh -e
+
+wget -q -O artifact_none.sh 'https://raw.githubusercontent.com/tidepool-org/tools/ci-changes/artifact/artifact.sh'
```
Comment (pazaan) [inline artifact.sh]: Why `artifact_none`? Isn't it producing a docker image (which is an artifact)?

Comment (derrickburns) [reply artifact.sh]: I will change the names in the next pass.  The name it cannot be is artifact.sh!

Comment (pazaan) [reply artifact.sh]: Ah, of course.

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@tidepool/viz",
-  "version": "1.3.0-weekly-to-bg-log.2",
+  "version": "1.3.0",
```
Comment (pazaan) [inline package.json]: Should this have a different version? @cbwebdevelopment

Comment (derrickburns) [reply package.json]: Good catch. I rebased on top of develop.  So, Clint's commits are not part of this now.

**Pull request merged**